src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issueadjournment/IssueAdjournmentNoticeAboutToSubmitHandler.java

```diff
@@ -152,11 +152,7 @@ private static State isCaseListable(SscsCaseData sscsCaseData) {
 
     private void clearAdjournmentTransientFields(SscsCaseData caseData) {
         if (isAdjournmentEnabled) {
```

Comment:
could we remove this flag assuming adjournment is always going to be enabled now
also, is there a unit test that covers this?
and we should probably remove the clearing that we're doing in the aboutToStart because that's obviously not working, so it's just misleading